corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js

```diff
@@ -696,6 +708,9 @@ hqDefine('app_manager/js/case-config-ui-advanced.js', function () {
                             return 'Lookup table tag required';
                         }
                     }
+                    if (!self.case_type()) {
+                        return 'Expected case type required';
```

Comment:
Will this mean we have to set these expected case types on all case management pages that currently have autoloads currently when we save (even if the change is to a different load action)?